src/Sulu/Bundle/TestBundle/Testing/ContainerTrait.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) Sulu GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Bundle\TestBundle\Testing;
-
-use Symfony\Component\DependencyInjection\ContainerInterface;
-
-trait ContainerTrait
-{
-    /**
-     * @internal The "getContainer" function is deprecated use self::$container variable instead.
-     *
-     * This function exist to keep compatibility to old tests.
-     */
-    protected static function getContainer(): ContainerInterface
```

Comment:
This is now provided by [Symfony itself](https://github.com/symfony/symfony/blob/d11e41be6cb23016894c564d5c2b9ef3357ecaa7/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L100) and conflicts with it so we needed to remove it.